sentences/sv/_common.yaml

```diff
@@ -51,14 +51,19 @@ expansion_rules:
   area: "{area}[e | me][t | ts | n | ns ]"
   vad: "(vad är | vad är det | vad är det för | vilken)"
   är: "(är det | är)"
-  brightness: "{brightness} [percent]"
+  brightness: "{brightness} [percent | procent]"
```

Comment:
should we keep the English version for percent? Seems like we might want to limit the scope to Swedish and as I understand it it would be possible for end users to add their own commands also.

Reply:
Fixed :)